### ISSUE TITLE
Improve accessibility of TMAP8

### ISSUE THREAD

**Issue by cticenhour** [idaholab/TMAP8#388]
## Reason
Currently there are 20 warnings associated with lack of alt-text for images in TMAP8. This should be improved to allow for better accessibility on the website.

## Design
Use the `alt-text` parameter for media on all figures within TMAP8 that do not use a caption. 

## Impact
Improve accessibility for those TMAP8 users who are using screen readers.
